src/js/view/controls/settings-menu.js

```diff
@@ -122,14 +127,15 @@ export function setupSubmenuListeners(settingsMenu, controlbar, model, api) {
             removePlaybackRatesSubmenu(settingsMenu);
             return;
         }
-
-        addPlaybackRatesSubmenu(
-            settingsMenu,
-            playbackRates,
-            model.setPlaybackRate.bind(model),
-            model.get('playbackRate'),
-            model.get('localization').playbackRates
-        );
+        if (!settingsMenu.getSubmenu('playbackRates')) {
```

Comment:
You shouldn't do this - we may need to replace an existing playback rate submenu with one containing different options. The way we do this in the settings menu is to completely replace the submenu.

Reply:
Updated